etna/datasets/internal_datasets.py

```diff
@@ -132,13 +158,13 @@ def load_dataset(
     if not _check_dataset_local(dataset_path) or rebuild_dataset:
         get_dataset_function(dataset_dir)
     if len(parts_) == 1:
```

Comment:
Why do we need this `if/else`. There is a lot of duplication for it and it can lead to some kind of errors. Can't we somehow unite the logic. For example, there will be only some `if/else` at the end that returns first element if `len(...) == 1`.